Added CV Screener

Areebah03 opened the pull request:
*Fixed #1409

<hr>

## Description 📜

This website show us details of different employees along with their photo, age, city, skills, language etc. We can can view different employee details by clicking next button. At the end of application website is reset to first employee. I have used html, css, javascript in this website.

Fixes #1409 

<hr>

## Type of change 📝


- [x] Add a new project



<hr>

## Domain of Contribution 📊



- [x] Project using HTML, CSS & JS


<hr>
 
## Checklist ✅


- [x] I have performed a self-review of my own code or work.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] I'm GSSoC'22 contributor


<hr>
 
## Screenshots / Gif (Recommended) 📸
![ss1](https://user-images.githubusercontent.com/85051124/167339781-81319fdc-92b5-4434-a5d8-e08ec4b8cca6.png)
![2](https://user-images.githubusercontent.com/85051124/167339802-a8e3db31-d77d-49e9-8577-d10d87567cb1.png)

### End of application
![3](https://user-images.githubusercontent.com/85051124/167339799-0fdfa46c-4f75-450d-b77d-ca045d055b57.png)

gurjeetsinghvirdee reviewed (changes requested):
write `project name`, `description`, `tech-stack` used and add demo video in `readme` file

Areebah03 commented:
https://user-images.githubusercontent.com/85051124/167342401-3616d7e1-3c15-45a0-9355-ba32986fd101.mp4

Areebah03 commented:
@gurjeetsinghvirdee I have added video and updated README.md file. I have written project name, description, tech-stack used in README.md file. Kindly view.
